libcaf_net/caf/net/datagram_transport.hpp

```diff
@@ -127,7 +125,7 @@ class datagram_transport : public datagram_transport_base<Factory> {
     return none;
   }
 
-  void write_packet(id_type id, span<buffer_type*> buffers) override {
+  void write_packet(id_type id, span<byte_buffer*> buffers) override {
```

Comment:
`caf/byte_buffer.hpp` is not included in this file.

Reply:
`caf/fwd.hpp` is included here though.

Reply:
We are accessing member functions of `byte_buffer`. Hence, a forward declaration is insufficient and we need the full type declaration.